pages.it/linux/radeontop.md

```diff
@@ -0,0 +1,20 @@
+# radeontop
+
+> Mostra Utilizzo di AMD GPUs.
+> Maggiori informazioni: <https://github.com/clbr/radeontop>.
+
+- Mostra utilizzo del AMD GPU:
```

Comment:
There is another example that talks about `select a specific GPU`, why you didn't add the concept of `default GPU` like the main page says?

Reply:
Fixed.